module/VuFind/src/VuFind/Db/Entity/AccessToken.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+/**
+ * Entity model for access_token table
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
+ */
+
+namespace VuFind\Db\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+use VuFind\Db\Feature\DateTimeTrait;
+
+/**
+ * Entity model for access_token table
+ *
+ * @category VuFind
+ * @package  Database
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
+ */
+#[ORM\Table(name: 'access_token')]
+#[ORM\Entity]
+class AccessToken implements AccessTokenEntityInterface
+{
+    use DateTimeTrait;
+
+    /**
+     * Unique ID.
+     *
+     * @var string
+     */
+    #[ORM\Column(name: 'id', type: 'string', length: 255, nullable: false)]
+    #[ORM\Id]
+    #[ORM\GeneratedValue(strategy: 'NONE')]
+    protected $id;
+
+    /**
+     * Token type.
+     *
+     * @var string
+     */
+    #[ORM\Column(name: 'type', type: 'string', length: 128, nullable: false)]
+    #[ORM\Id]
+    #[ORM\GeneratedValue(strategy: 'NONE')]
+    protected $type;
+
+    /**
+     * User.
+     *
+     * @var UserEntityInterface
+     */
+    #[ORM\JoinColumn(name: 'user_id', referencedColumnName: 'id')]
+    #[ORM\ManyToOne(targetEntity: \VuFind\Db\Entity\User::class)]
```

Comment:
Should the `targetEntity` attribute refer to the interface instead of the actual class (here and elsewhere)?

Reply:
If that's the case, then let's just leave it there. I've no idea how Doctrine instantiates the class, that's why it was just a question.

Reply:
As @aleksip has been working on updating these, I'll mark this resolved. :)